config/urls.py

```diff
@@ -33,6 +32,7 @@
     url(r"^messages/", include("bootcamp.messager.urls", namespace="messager")),
     url(r"^qa/", include("bootcamp.qa.urls", namespace="qa")),
     url(r"^search/", include("bootcamp.search.urls", namespace="search")),
+    url("", include("bootcamp.users.urls", namespace="users")),
```

Comment:
Don't remove the line and add it in a different place unless necessary, this changes brakes the traceability for future references.

Reply:
How would you do this? So users URL follow the pattern domain/user instead of domain/users/user.

Reply:
Oh no, I'm actually referring to you deleting the URL from line 20 and creating it again in line 35. I actually understand why you want to remove the `/user/` part, and I have no issue with that.